docker-bake.hcl

```diff
@@ -30,99 +34,166 @@ group "default" {
 }
 
 group "all" {
-  targets = ["node-group", "disperser-group", "retriever", "churner"]
+  targets = ["node-group", "batcher", "disperser", "encoder", "retriever", "churner", "dataapi", "traffic-generator"]
 }
 
 group "node-group" {
   targets = ["node", "nodeplugin"]
 }
 
-group "disperser-group" {
-  targets = ["batcher", "disperser", "encoder"]
-}
-
+# Github public releases
 group "node-group-release" {
   targets = ["node-release", "nodeplugin-release"]
 }
 
-# DOCKER METADATA TARGET
-# See https://github.com/docker/metadata-action?tab=readme-ov-file#bake-definition
+# Github CI builds
+group "ci-release" {
+  targets = ["node-group", "batcher", "disperser", "encoder", "retriever", "churner", "dataapi"]
+}
+
+# Internal devops builds
+group "internal-release" {
+  targets = ["node-internal", "batcher-release", "disperser-release", "encoder-release", "retriever-release", "churner-release", "dataapi-release", "traffic-generator-release"]
```

Comment:
Why is it node-internal and not node-release?

Reply:
node-release was already used for public node releases and is incompatible with internal release targets.

Reply:
I'll rename `node-internal-release`

Reply:
also confused. is it because node-release is already something separate which builds with the arm target?

Should we call all the other internal-release targets xxx-internal instead of release then? Especially if those things are only internal, why call them release?

Reply:
> also confused. is it because node-release is already something separate which builds with the arm target?

it's because the existing `node-release` is incompatible with out internal release tagging.

```
target "node-release" {
  inherits = ["node", "_release"]
  tags     = ["${REGISTRY}/${REPO}/opr-node:${BUILD_TAG}"]
}
```

```
target "node-internal-release" {
  inherits = ["node"]
  tags       = ["${REGISTRY}/eigenda-node:${BUILD_TAG}",
                "${REGISTRY}/eigenda-node:${GIT_SHA}",
                "${REGISTRY}/eigenda-node:sha-${GIT_SHORT_SHA}",
               ]
}
```

> Should we call all the other internal-release targets xxx-internal instead of release then? Especially if those things are only internal, why call them release?

Definitely agree that "release" is way overloaded. I'll rename